src/pages/People.tsx

```diff
@@ -0,0 +1,155 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+import { Loader } from '../components/Loader';
+
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+type Props = {
+  selectedPerson: string | null,
+};
+
+enum ERROR {
+  NONE,
+  EMPTYARRAY,
+  OTHERERROR,
+}
+
+export const People: React.FC<Props> = ({ selectedPerson }) => {
+  const [error, setError] = useState<ERROR>(ERROR.NONE);
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const findMother = (person: Person) => {
+    if (person.motherName) {
+      return people.find((p) => p.name === person.motherName);
+    }
+
+    return undefined;
+  };
+
+  const findFather = (person: Person) => {
```

Comment:
This func is similar to `findMother` function, it's better to create reusable `findParents`